project/src/main/java/com/google/sps/servlets/QueryServlet.java

```diff
@@ -38,6 +38,8 @@
 @WebServlet("/query")
 public final class QueryServlet extends HttpServlet {
 
+  private static final long serialVersionUID = 1L;
```

Comment:
I don't really think you need that...

Reply:
There is an annoying warning that appears as long as we don't have that haha. I know it's not needed, that's why it wasn't there so far. It's just annoying to work with, and I think it's harmless to add this.

Reply:
Ack.